StudenSchool_DO/HWALL/HW3/Provider/Store.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Provider;
+
+public abstract class Store
```

Comment:
It's kind of overkill. Using abstract class only for sharing connection string is just too much